internal/database/migration_20230126_0000/20230126-0000.go

```diff
@@ -40,7 +40,7 @@ type Device struct {
 // Organization contains Users and their Devices
 type Organization struct {
 	Base
-	Users       []*User `gorm:"many2many:user_organization;"`
+	Users       []*User `gorm:"many2many:user_organizations;"`
```

Comment:
@dave-tucker  GORM can't query it if it isn't pluralized. It will come back with a `user_organizations` table not found unless we implement this interface is the only workaround I see.  https://gorm.io/docs/conventions.html#Pluralized-Table-Name

Reply:
💩 Yeah this is going to suck.
We need a new DB migration to rename that table then 😢

Reply:
@dave-tucker Done!